pkg/kube/util.go

```diff
@@ -209,12 +209,17 @@ func SetRestDefaults(config *rest.Config) *rest.Config {
 	return config
 }
 
-// CheckPodTermina returns true if the pod's phase is terminal (succeeded || failed)
+// CheckPodTerminal returns true if the pod's phase is terminal (succeeded || failed)
 // usually used to filter cron jobs.
 func CheckPodTerminal(pod *corev1.Pod) bool {
 	return pod.Status.Phase == corev1.PodFailed || pod.Status.Phase == corev1.PodSucceeded
 }
 
+// CheckPodRunning returns true if the pod's phase is Running || Succeeded.
+func CheckPodRunning(pod *corev1.Pod) bool {
+	return pod.Status.Phase == corev1.PodRunning || pod.Status.Phase == corev1.PodSucceeded
```

Comment:
because I was trying to hard to invent a new pod state term for `(Running || Succeeded)` here, which was probably misguided.

`CheckPodActive` or something also works.